.github/workflows/documentation_examples_check.yml

```diff
@@ -1,5 +1,14 @@
 name: documentation_examples_check
-on: push
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```

Comment:
I think - on push should be sufficient, let's remove the pull request trigger. In the past when I've tried keeping both triggers we get unnecessary runs

Reply:
Fixed